shared/src/main/java/uk/gov/di/authentication/shared/services/DynamoMfaMethodsService.java

```diff
@@ -87,4 +89,32 @@ private List<MfaMethodData> getMfaMethodsForNonMigratedUser(
             return List.of();
         }
     }
+
+    @Override
+    public MfaMethodData postMfaMethod(String email, MfaMethodCreateRequest request)
```

Comment:
I think we should change the language here as this is about the database rather than HTTP layer, so 'add' instead of 'post'. Maybe 'addBackupMfaMethod'?